### PR TITLE
Redirect the user to login page on search route

### PR DESCRIPTION
**Pull request by jonathanyeong**
https://github.com/scirate/scirate/issues/390

## Description
This PR redirects the user to the login page on the search route. Unless they have been signed in.

### CODE REVIEW

File: app/controllers/papers_controller.rb [hunk 1]
```diff
@@ -33,6 +33,8 @@ def __quote(val)
   end
 
   def search
+    redirect_to login_path unless signed_in?
```
Comment (jonathanyeong) [inline app/controllers/papers_controller.rb:36]: Wondering if I should add a `notice: "Please sign in."` to this.

Comment (silky) [reply app/controllers/papers_controller.rb:36]: i think it doesn't hurt to add it, if someone happens to be visiting the url directly :)

**Comment by silky** [comment]
thanks for working on this @jonathanyeong  :)

**Comment by silky** [comment]
thanks again @jonathanyeong !

**Comment by jonathanyeong** [comment]
Glad I can help!

**Pull request merged**